packages/lb-annotation/src/utils/tool/AttributeUtils.ts

```diff
@@ -254,14 +255,36 @@ export default class AttributeUtils {
     baseIcon: any,
   ) {
     if (attributeConfigurable === true) {
-      // 含有属性配置
+      // Open attribute configuration
       const attributeIndex = (attributeList?.findIndex((i: any) => i?.value === attribute) % COLORS_ARRAY.length) + 1;
+      const color = attributeList?.find((i: any) => i.value === attribute)?.color;
+      if (color) {
+        const fill = ToolStyleUtils.getToolColorList(color)?.invalid?.stroke;
+        const getHexColor = ToolStyleUtils.rgbaToHex(fill);
+        return this.changeIconAttribute({ fill: getHexColor });
+      }
       return ICON_ARRAY[attributeIndex];
     }
 
     return baseIcon;
   }
 
+  public static changeIconAttribute({
```

Comment:
Please update function name from `changeIconAttribute` to `generateIconAttribute`